src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -115,9 +113,17 @@ func Init() (*Config, error) {
 			return
 		},
 	}
-	initial, err := m.Start()
+	initial, err := m.Init()
 	var cfg *Config
 	if err == nil {
+		er := pubsub.Sub(&geo.City{}, func(city *geo.City) {
+			log.Debugf("Got location: %v", city.Country.IsoCode)
+			m.StartPolling()
```

Comment:
Cool -- nice idea. I just added that in yamlconf itself actually, which seems a bit more bulletproof. Good catch.